docs/user/00-30-runtime-agent-overview.md

```diff
@@ -1,8 +1,34 @@
 # Runtime Agent
 
-Runtime Agent is a part of the Application Connector module that connects to [Compass](https://github.com/kyma-incubator/compass). It is managed by the Application Connector module operator. It is an integral part of every Kyma Runtime in the [Compass mode](README.md), and it fetches the latest configuration from Compass. It also provides Runtime-specific information that is displayed in the Compass UI, such as Runtime UI URL, and it provides Compass with Runtime configuration, such as Event Gateway URL, that should be passed to an Application. To learn more, read the section on [configuring the Runtime](technical-reference/07-20-configuring-runtime.md).
+> [!WARNING]
+> Runtime Agent is currently not integrated into the Application Connector module. Proceed with caution and consult the respective documentation for the Application Connector module's current configuration and functionality. 
```

Comment:
Could you change `Runtime Agent` to `Compass Runtime Agent` (in all warnings or crucial parts) to fully describe its purpose?